recipes/azure-mgmt-core/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please delete the comments throughout the recipe if they are from the sample and not directly commenting on this particular recipe.

Reply:
I have deleted it